storehaus-elasticsearch/src/main/scala/com/twitter/storehaus/elasticsearch/ElasticSearchStringStore.scala

```diff
@@ -33,15 +33,13 @@ import scala.collection.breakOut
  */
 
 object ElasticSearchStringStore {
-
-  def apply(index: String,
-            tipe: String,
-            client: Client):ElasticSearchStringStore = new ElasticSearchStringStore(index, tipe, client)
+  def apply(index: String, tipe: String, client: Client): ElasticSearchStringStore =
+    new ElasticSearchStringStore(index, tipe, client)
 }
 
-class ElasticSearchStringStore(private val index: String,
-                               private val tipe: String, //tipe -> type since type is a reserved keyword
-                               private val client: Client) extends Store[String, String] with QueryableStore[SearchRequest, String] {
+class ElasticSearchStringStore(
+  private val index: String, private val tipe: String, private val client: Client
+) extends Store[String, String] with QueryableStore[SearchRequest, String] {
 
   private lazy val futurePool = FuturePool.unboundedPool
```

Comment:
as a TODO for us in the future: this is not a good pattern. `FuturePool` should be passed by constructors (often implicit).